searchlib/src/vespa/searchlib/queryeval/executeinfo.h

```diff
@@ -0,0 +1,29 @@
+// Copyright 2019 Oath inc. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
+
+#pragma once
+
+namespace search::queryeval {
+
+    /**
+     * Holds information about how query will be executed and how large part of corpus will pass through.
+     * @author baldersheim
+     */
+class ExecuteInfo {
+public:
+    ExecuteInfo() : ExecuteInfo(false, 1.0) { }
```

Comment:
consider using 0.0 as default (is this only needed by tests?)

Reply:
Why 0.0 ? 1.0 sounds like a better(more conservative) value.
Only used in tests.

Reply:
reason for comment: false is the default for bool and 0.0 is the default for double.
I think it would be better to just remove it if possible.